keyboards/irene/readme.md

```diff
@@ -0,0 +1,18 @@
+# Irene
+
+![Irene](https://i.imgur.com/p4bWtKDl.jpeg)
+
+The Irene is a polycarbonate top-mount keyboard featuring the Alice layout inspired by TGR's Police and Jaxxstatic's Rukia. The name came from a Kpop idol Irene from Red Velvet.
+
+* Keyboard Maintainer: [Andrei Collado](https://github.com/colladoandrei)
+* Hardware Supported: ATmega32u4
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?

Reply:
Thanks! Added.